units/commit_queue.go

```diff
@@ -270,6 +270,35 @@ func (j *commitQueueJob) TryUnstick(ctx context.Context, cq *commitqueue.CommitQ
 		return
 	}
 
+	mergeTask, err := task.FindMergeTaskForVersion(nextItem.Version)
+	if err != nil {
+		j.AddError(errors.Wrap(err, "unable to find merge task for version"))
+	}
+	if mergeTask != nil {
+		// check that the merge task can run. Assume that if we're here the merge task
+		// should in fact run (ie. has not been dequeued due to a task failure)
+		if !mergeTask.Activated {
+			grip.Error(message.Fields{
+				"message": "merge task is inactive",
+				"project": mergeTask.Project,
+				"task":    mergeTask.Id,
+				"source":  "commit queue",
+				"job_id":  j.ID(),
+			})
+			j.AddError(model.SetActiveState(mergeTask, evergreen.APIServerTaskActivator, true))
+		}
+		if mergeTask.Blocked() {
```

Comment:
If the merge task is blocked, shouldn't we unstick the queue by removing the item? Or is this something you never expect to happen so we want to be notified if it ever does

Reply:
more the latter. I don't want to remove it because it's probably something that should be merged if it's still in the queue